.github/ISSUE_TEMPLATE/repo-create.md

```diff
@@ -13,15 +13,36 @@ e.g. cloud-provider-foo
 ### Which Organization should it reside
 e.g. kubernetes-sigs
 
-### Who should be provided access to this repository
-e.g (at)example_user should have write access
+### If not a staging repo, who should have admin access
+e.g. alice, bob
+
+### If not a staging repo, who should have write access
+e.g. chris, dianne
+
+### If a new repo, who should be listed as approvers in OWNERS
+e.g. alice, bob
+
+### If a new repo, who should be listed in SECURITY_CONTACTS
+e.g. alice, bob
+
+### What should the repo description be
+by default we will set it to "created due to (link to this issue)"
```

Comment:
I don't like using the link as a default description, but not strongly enough to actually hold this up.